### ISSUE TITLE
508 compliance issues (Accessibility) Try It panel on the API Details page does not have the correct minimum contrast ratio of 4.5:1 against its background

### ISSUE THREAD

**Issue by christopherwoodland** [Azure/api-management-developer-portal#1385]
## Bug description

Our client has noticed the following accessibility concerns and asked that we submit these concerns to the product team for resolution. These issues violate the 508 compliance, which the customer is bound to.  I am willing to aid in any way to bring these concerns to resolution. Please let me know if additional context or information is required to help move these concerns along.  I can be reached internally via teams or email cwoodland@microsoft.com
Thank you in advance!

1. The Try It panel on the API Details page does not have the correct minimum contrast ratio of 4.5:1 against its background. Specifically, the blue text in the screenshot below. Also on the API Details page, when viewing the “Raw Schema View” for an endpoint’s responses, the formatting characters, i.e., brackets and commas, do not meet the minimum contrast ratio.
 
![image](https://user-images.githubusercontent.com/16299417/125957341-e55fbe30-87b9-4fb9-810f-21ffb66a9348.png)
![image](https://user-images.githubusercontent.com/16299417/125957365-7be7b226-670d-4318-8e91-be51d2affa83.png)

#1383
#1384

**Comment by christopherwoodland** [comment]
@jdm4344  Perfect!  I will reference and close these issues.

#1374, #1375, #1378, #1379, #1380.
